src/pages/HomePageBoilerplate.js

```diff
@@ -0,0 +1,21 @@
+import React, { Component } from 'react';
+
```

Comment:
I'd vote in favor of removing this, as with all of the example files

Reply:
Probably, but maybe in another PR

Reply:
Fair point, new issue?

Reply:
Tracked here: https://github.com/NIAEFEUP/nijobs-fe/issues/17